tests/proofs/simple-arithmetic-spec.k

```diff
@@ -24,17 +24,17 @@ module SIMPLE-ARITHMETIC-SPEC
 
     rule <k> block [ .ValTypes ]
                  ( loop [ .ValTypes ]
-                     (local.get 0)
-                     (local.get 1)
-                     (i32.add)
-                     (local.set 1)
-                     (local.get 0)
-                     (i32.const 1)
-                     (i32.sub)
-                     (local.tee 0)
-                     (i32.eqz)
-                     (br_if 1)
-                     (br 0)
+                     local.get 0
+                     local.get 1
+                     i32.add
+                     local.set 1
+                     local.get 0
+                     i32.const 1
+                     i32.sub
+                     local.tee 0
+                     i32.eqz
+                     br_if 1
+                     br 0
```

Comment:
@hjorthjort The reason why we cannot write `(PlainInstr)` here is because it is inside a rule. Inside a rule an empty string cannot be parsed as a empty list, so that a plain instruction for example `( i32.eqz )`, cannot be matched with the syntax `(PlainInstr Instr)`.
Now we add a new syntax `( PlainInstr )`, and now we can use `(PlainInstr)` directly in rules of prove specs.